scraper/src/github-scraper/discussion.ts

```diff
@@ -50,44 +50,42 @@ async function fetchGitHubDiscussions(
   org: string,
   endDate: Date,
   startDate: Date,
-  cursor = null,
 ) {
-  const variables = {
-    org,
-    cursor,
-  };
-  for await (const response of octokit.graphql.paginate.iterator(
-    query,
-    variables,
-  )) {
-    const repositories = await response.organization.repositories.edges;
-    type repo = (typeof repositories)[0];
+  const iterator = octokit.graphql.paginate.iterator(query, { org });
+
+  let Discussions: { repository: string; discussion: Discussion }[] = [];
```

Comment:
Please use proper naming conventions! `discussions`

Reply:
Used `discussionsList`  for more clarity

Reply:
perfect